pkgs/development/python-modules/mapbox-earcut/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, setuptools
+, numpy
+, pybind11
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mapbox-earcut";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "skogler";
+    repo = "mapbox_earcut_python";
+    rev = "257c2c82819acf72b2b757f200c160e55b9427fc";
```

Comment:
Why are we not fetching https://github.com/skogler/mapbox_earcut_python/releases/tag/v0.12.10?

Reply:
Because this repo does not receive a lot of commits, but after v0.12.10 release there have been some substantial commits that i would not want to miss. Specifically switching from cmake to complete setuptools build-system.

Reply:
Then we need to change the version number to unstable-XXXX-XX-XX where the XXXX-XX-XX is the date of the commit.

Reply:
ah alright! fixed.